zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterRunningProcess.java

```diff
@@ -80,6 +88,10 @@ public Void call(RemoteInterpreterService.Client client) throws Exception {
         } catch (Exception e) {
           logger.warn("ignore the exception when shutting down interpreter process.", e);
         }
+
+        // Shutdown connection
+        shutdown();
+        logger.info("Remote process of interpreter group: " + getInterpreterGroupId() + " is terminated");
```

Comment:
Could you please make the logger 'public static final' and avoid the creation of objects if the logger is disabled for the INFO level ( `LOGGER.info(String format, Object... arguments);`).

Reply:
fixed